apps/client/src/types/pokemons.ts

```diff
@@ -0,0 +1,4 @@
+export type BasicPokemon = {
+  name: string;
+  id: string;
+};
```

Comment:
I'd do something like this:

```suggestion
export type BasicPokemon = Pick<PokemonSummary, 'name'> & {
  id: string;
};
```